app/directives/proposals/proposals.directive.js

```diff
@@ -133,27 +199,14 @@ angular.module('dashboard')
                     }
                     var draft = createDraft(type.value);
                     self.proposals.splice(0, 0, draft);
+                    countProposals();
                 }
                 else {
                     $log.error('dbProposals: addProposal parameter invalid');
                 }
-                countProposals();
             }
 
-            function removeProposal(guid) {
-                var search = angular.isArray(self.proposals);
-                for (var index = self.proposals.length + CONST.NOTFOUND; search && index > CONST.NOTFOUND; index--) {
-                    var localProposal = self.proposals[index];
-                    if (angular.equals(guid, localProposal.proposalGuid)) {
-                        self.proposals.splice(index, 1);
-                        search = false;
-                    }
-                }
-                checkProposals();
-                countProposals();
-            }
-
-            self.delete = function(data) {
+            self.delete = function (data) {
```

Comment:
Should this be `del`. Compare to proposals.directive.html
 `on-delete` attribute value